draft-ietf-quic-transport.md

```diff
@@ -3674,7 +3707,37 @@ Token Length:
 
 Token:
 
-: The value of the token.
+: The value of the token that was previously provided in a Retry packet or
+  NEW_TOKEN frame.
+
+Length:
+
+: The length of the remainder of the packet (that is, the Packet Number and
+  Payload fields) in bytes, encoded as a variable-length integer
+  ({{integer-encoding}}).
+
+Packet Number:
+
+: The packet number field is 1 to 4 bytes long. The packet number has
+  confidentiality protection separate from packet protection, as described in
+  Section 5.4 of {{QUIC-TLS}}. The length of the packet number field is encoded
+  in the plaintext packet number. See {{packet-encoding}} for details.
+
+Payload:
+
+: The payload of the packet.
+
+### Initial Packet {#packet-initial}
```

Comment:
I think that we should use h2 for this.  The section is a mix of "here is a common format" and "here is a specific instance of a common format", but I think that's tolerable.  We would then have something like:

```
17. Packet Formats
    17.1. Packet Number Encoding and Decoding
    17.2. Long Header Packets <-- note the extra 's'
    17.3. Version Negotiation Packet
    17.4. Retry Packet
    17.5. Extended Long Header Packets
    17.6. Initial Packet
        17.6.1. Abandoning Initial Packets
        17.6.2. Starting Packet Numbers
    17.7. 0-RTT Protected Packet <-- extra 'Packet'
        17.7.1. 0-RTT Packet Numbers
    17.8. Handshake Packet
    17.9. Short Header Packets <-- extra 's'
```

Does that work?

Reply:
I understand the desire not to move subsections below h3, but the mix of "format" and "instance" really just doesn't flow for me.  But it does kind of match the rest of the document with the "grouped sections" motif, so maybe it's at least consistent.

(It's already contorted enough that VN is a long header packet, but not from this version and therefore absent from the type table.)